cli/package.json

```diff
@@ -5,7 +5,7 @@
     "xstate": "./bin/index.js"
   },
   "scripts": {
-    "prepare": "esbuild src/bin.ts --outfile=bin/index.js --bundle --platform=node --external:./node_modules/* --external:../node_modules/*"
+    "prepare": "esbuild src/bin.ts --outdir=bin --platform=node --format=cjs"
```

Comment:
The previous script was not correct because it left relative paths to node_modules in the output